docs/blog/_posts/2019-12-20-21th-dotty-milestone-release.md

```diff
@@ -0,0 +1,378 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.21.0-RC1
+author: Aggelos Biboudis
+authorImg: /images/aggelos.png
+date: 2019-12-21
+---
+
+Greetings and we wish you Merry Christmas 🎄!
+
+We are excited to announce 0.21.0-RC1 of Scala 3. In this version we add support
+for non-nullable reference types and a flow-sensitive analysis that work
+complementary. We present new syntax for given extensions, matches and pattern
+matching over quotes. We are also happy to announce that SemanticDB extraction
+is now supported!
+
+# Feature Complete!
+
+This release is a HUGE milestone for us, for Dotty, for Scala 3, for our community. Since that
+[initial commit](https://github.com/lampepfl/dotty/commit/90962407e72d88f8f3249ade0f6bd60ff15af5ce)
+on the 6th December of 2012 when the only feature was the basic structure of a
+compiler based on the DOT calculus, we have come a long way.
+
+7 years and 20k commits later we are happy to announce that we are now _feature-complete_ for Scala 3.
+This means that with this release we stop adding new features and we focus on:
+
+- bug fixing and general quality assurance towards the final release 🐛
+- performance engineering 🏎️
+- documentation improvements 📕
+- education 👨‍🏫
+
+Being feature complete does not mean that every detail of Scala-3 is cast in
+stone yet. Some details can still change, or be enabled conditionally, or even
+be dropped entirely. That will depend on the additional experience we gain over
+the next months, in particular the feedback we receive from the community and
+the SIP committee. But the envelope of what will (most likely) be in Scala 3 is
+now defined and implemented.
+
+For an overview of the feature envelope that Scala 3 carries you can read our
+[Overview](https://dotty.epfl.ch/docs/reference/overview.html) page.
+For a more detailed discussion on the transition to Scala 3 you can read the
+[Scala 2 roadmap update: The road to Scala 3](https://www.scala-lang.org/2019/12/18/road-to-scala-3.html).
+
+### Community-build
+
+Being feature complete doesn't mean that development slows down. On the contrary!
+It means that we can now put the Scala 3 compiler under heavy load, getting it
+ready for industrial strength applications. At the moment we have 23
+projects on our community projects and we expect this number to go up!
+
+> https://github.com/lampepfl/dotty/tree/master/community-build/community-projects
+
+This project contains tests to build and test a corpus of open sources Scala 2.x
+projects against Scala 3.
+
+To run the community-build on a local machine from the main Dotty repository, first fetch all the git
```

Comment:
https://www.scala-lang.org/2019/12/18/road-to-scala-3.html

Reply:
You mean to add a link to the blogpost here?

Reply:
This is what I did 👍